src/Parser/ESLintParser.ts

```diff
@@ -13,6 +13,18 @@ export class ESLintParser extends Parser {
       const logs = JSON.parse(content) as ESLintLog[];
       return logs
         .filter((log) => log.messages.length !== 0)
+        .concat(
+          logs
```

Comment:
:warning: 'content' is defined but never used. (rule: @typescript-eslint/no-unused-vars)